stardis/radiation_field/opacities/opacities_solvers/tests/test_broadening.py

```diff
@@ -613,3 +614,13 @@ def test_calc_gamma_van_der_waals_cuda_wrapped_sample_cuda_values(
         calc_gamma_van_der_waals_cuda(*map(cp.asarray, arg_list)),
         calc_gamma_van_der_waals_sample_values_expected_result,
     )
+
+
+def test_rotational_broadening(example_stardis_output):
+    actual_wavelengths, actual_fluxes = rotation_broadening(
+        20 * u.km / u.s,
+        example_stardis_output.lambdas,
+        example_stardis_output.spectrum_lambda,
+        v_rot=0 * u.km / u.s,
```

Comment:
Currently this test just makes sure that when the rotation velocity is zero it returns the same output, there should probably be one that tests nonzero v_rot

Reply:
Added a small regression test. It's hardcoded since we don't have regression data set up yet, but that's something to implement and revisit.